src/connection/ConnectionOptionsReader.ts

```diff
@@ -138,6 +138,13 @@ export class ConnectionOptionsReader {
                 });
                 Object.assign(connectionOptions, { migrations: migrations });
             }
+            if (options.type === "sqlite") {
```

Comment:
It does not look like a good place to handle it. You need to handle this option in sqlte driver itself.